github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -821,6 +821,99 @@ periodics:
     volumes:
     - emptyDir: {}
       name: audit
+- name: periodic-kubevirt-e2e-test-S390X
+  annotations:
+    testgrid-dashboards: kubevirt-periodics
+    testgrid-days-of-results: "60"
+  cron: "30 4,12,20,00 * * *"
+  decorate: true
+  cluster: kubevirt-prow-workloads
+  decoration_config:
+    timeout: 4h
+    grace_period: 5m
+  labels:
+    preset-podman-in-container-enabled: "true"
+    preset-docker-mirror-proxy: "true"
+    preset-shared-images: "true"
+    preset-bazel-cache: "true"
+    preset-kubevirtci-quay-credential: "true"
+  extra_refs:
+  - org: kubevirt
+    repo: kubevirt
+    base_ref: main
+    work_dir: true
+  - org: kubevirt
+    repo: project-infra
+    base_ref: main
+  reporter_config:
+    slack:
+      job_states_to_report: []
+  spec:
+    nodeSelector:
+      type: bare-metal-external
+    containers:
+      - image: quay.io/kubevirtci/bootstrap:v20240607-febc467
+        env:
+          - name: TARGET
+            value: "external-wg-s390x"
```

Comment:
What is this target? I am not familiar with it? Is this needed?

Reply:
Hi @brianmcarey We are running the e2e tests using labels. The label is `wg-s390x` when we pass the TARGET as `external-wg-s390x`. it will extract the label wg-s390x and run the e2e tests on the external cluster.
These changes Iam adding them as a part of the e2e-tests enablement PR. https://github.com/kubevirt/kubevirt/pull/12516/files#diff-ce25e369a9df9edc525425cc144fc8c40599fff3cc1e5058b64336d71af67162R100-R427

Reply:
Ok but it is it not enough just to set `KUBEVIRT_PROVIDER=external` as you have done below. Then this [if statement](https://github.com/kubevirt/kubevirt/pull/12516/files#diff-ce25e369a9df9edc525425cc144fc8c40599fff3cc1e5058b64336d71af67162R100-R427) is not required?

Reply:
Hi @brianmcarey No, it is not enough, actually how they are running e2e tests with labels is they pass the  label+provider in the TARGET and they compare with the label and store the provider in the KUBEVIRT_PROVIDER variable and label in the label_filter variable. For ex: TARGET=k8s-1.29-sig-monitoring, KUBEVIRT_PROVIDER=k8s-1.28 and label_filter=sig-monitoring. In order to maintain the consistency I have done the same for the s390x e2e tests.

Reply:
Ok but we could just set `TARGET=external` and then there is no need to add that if statement in your kubevirt PR - we don't need to add a new target in automation/test.sh

Reply:
Hi @brianmcarey  ok, now I got it, so you are saying to put the `TARGET=external` and then provide the `KUBEVIRT_LABEL_FILTER=wg-s390x.` this way we can avoid the if statement and the new target in automation/test.sh ?